cmds/cmd_automod.py

```diff
@@ -695,4 +690,4 @@ async def add_joinrule(message: discord.Message, args: List[str], client: discor
             },
     }
 
-version_info: str = "1.2.12"
+version_info: str = "1.2.13-DEV"
```

Comment:
versioning in a couple files is -DEV

Reply:
don't know if this is intended or not

Reply:
yeah thats a TODO before i merge, same as the changelog

Reply:
will leave this convo open until ive pushed that

Reply:
cool